src/protobuf/BUILD

```diff
@@ -0,0 +1,3 @@
+protobuf_sources(
+  grpc=True,
+)
```

Comment:
I'm a little confused having both `src/protobuf/service.proto` and then a folder below it called `protobuf_example`. I would expect these two to be related, but it doesn't look like they are.

How about instead having no sources in the top-level `src/protobuf`, and then having something like `src/protobuf/simple_example` and `src/protobuf/grpc_example`? That would make it clearer that there is no coupling.

Reply:
Reorganized the protobuf sources. Please review the new arrangement.